.github/workflows/ci.yml

```diff
@@ -0,0 +1,46 @@
+name: Result-Dist CI Workflow
+
+on: [pull_request]
+
+jobs:
+  distributor_build:
+    
+    runs-on: ubuntu-latest
+    
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+    
+      - name: Distributor Build
+        uses: ballerina-platform/ballerina-action@master
```

Comment:
Here we depend on some third-party repo and it's ballerina version changes frequently. Ideally, org ECLK should maintain that docker

Reply:
Switched to 1.2.4. I guess we should unify how we specify the ballerina version here and for the Dockerfile.